jobs/models.py

```diff
@@ -8,36 +8,65 @@ class JobType(models.TextChoices):
     INTERNSHIP = '3', 'Internship'
 
 
+class Location(models.TextChoices):
+    Tel_Aviv = '1', 'Tel Aviv'
+    Jerusalem = '2', 'Jerusalem'
+    Haifa = '3', 'Haifa'
+
+
 # Jobs Model
 class Job(models.Model):
     title = models.CharField(max_length=100)
     description = models.CharField(max_length=255)
-    location = models.CharField(max_length=100)
+    location = models.CharField(
+        max_length=15,
+        choices=Location.choices,
+        default=Location.Tel_Aviv,
+        blank=True, null=True
+    )
     job_type = models.CharField(
         max_length=15,
         choices=JobType.choices,
         default=JobType.PART_TIME,
         blank=True, null=True
     )
     company_name = models.CharField(max_length=255)
-    company_description = models.CharField(max_length=255)
     post_until = models.DateField()
     is_active = models.BooleanField()
+    marked_count = models.IntegerField(default=0)
+    apply_link = models.URLField(unique=True, null=True)
 
     # Function to create a new job
     @classmethod
-    def create_job(cls, title, description, location, job_type, company_name,
-                   company_description, post_until, is_active):
+    def create_job(cls, title, description, location, job_type,
+                   company_name, post_until, is_active, marked_count, apply_link):
         job = cls(title=title, description=description, location=location,
                   job_type=job_type, company_name=company_name,
-                  company_description=company_description, post_until=post_until, is_active=is_active)
+                  post_until=post_until, is_active=is_active, marked_count=marked_count, apply_link=apply_link)
         job.save()
         return job
 
+    def del_job(self):
+        try:
+            self.job.delete()
+        except Job.DoesNotExist:
+            return False
+        return True
+
     # Function to reduce the length of description
     def snippet_description(self):
-        return self.description[:25] + '...'
+        snippedDes = self.description[:25] + '...'
+        return snippedDes
 
     # Jobs title to string - return the title of the job
     def __str__(self) -> str:
         return self.title
+
+    # Jobs title to string - return the title of the job
+    def get_description(self) -> str:
+        return self.description
+
+    @staticmethod
+    # Function to search job by title
+    def get_jobs_by_title(title):
```

Comment:
Need to add a test for this function, I think build two fixture with same title (example: software developer) then run the functions and make sure you get both jobs

Reply:
The function is not working. Do you know any other way to search for a set of objects by attribute? I looked online and couldn't find something else that works.

Reply:
I tried a function called "def test_get_jobs_same_title" but I don't think I am close to writing it write. Do you have any suggestions? the function is commented bellow in line 51 in jobs/tests.py

Reply:
@assafg95 since this does have to be by Demo 3 (We are only going to complete signup and login) I suggest you minimize this PR to the updated model, model function and relevant test which are working. If we will have time left before Demo 3 we will add the relevant function to the search feature. If something is not clear PM me via slack.

Reply:
Looks good. Fixed

Reply:
when fixed, you can click the resolved button